src/App.tsx

```diff
@@ -1,14 +1,61 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './App.scss';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { StatusFilter } from './helpers/StatusFilter';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loader, setLoader] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+  const [todoFilter, setTodoFilter] = useState<StatusFilter>(StatusFilter.All);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  const visibleTodos = todos.filter(todo => {
+    const formattedTitle = todo.title.toLowerCase().trim();
+    const formattedInputValue = inputValue.toLowerCase().trim();
```

Comment:
you do `.toLowerCase().trim()` twice here. So let's probably create some helper function like `normalizeString()` instead?